src/main/java/org/cloudburstmc/api/CloudburstApi.java

```diff
@@ -0,0 +1,20 @@
+package org.cloudburstmc.api;
+
+public abstract class CloudburstApi {
+    private static CloudburstApi instance;
+
+    public static void set(CloudburstApi instance) {
+        if (CloudburstApi.instance != null) {
+            throw new IllegalStateException("Internal has already been set");
+        }
+
+        CloudburstApi.instance = instance;
+    }
+
+    public static CloudburstApi get() {
+        return instance;
+    }
+
+    public abstract Server getServer();
+
```

Comment:
@SupremeMortal what are your thoughts on this schema for being able to statically get a `Server` instance? 
Or would you rather use `@Inject`?

Reply:
I'm still for dependency injection.